dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -246,6 +248,59 @@ def test_add_profilers(self, *mocks):
         self.assertEqual("multiple files", merged_profile.encoding)
         self.assertEqual("multiple files", merged_profile.file_type)
 
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "ColumnPrimitiveTypeProfileCompiler"
+    )
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnStatsProfileCompiler")
+    @mock.patch("dataprofiler.profilers.profile_builder." "ColumnDataLabelerCompiler")
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder.DataLabeler", spec=StructuredDataLabeler
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._update_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder."
+        "StructuredProfiler._merge_correlation"
+    )
+    @mock.patch(
+        "dataprofiler.profilers.profile_builder." "StructuredProfiler._update_chi2"
+    )
+    def test_add_profiles_null_count_not_enabled(self, *mocks):
+        profiler1_options_null_count = ProfilerOptions()
+        profiler1_options_null_count.set(
+            {"row_statistics.null_count.is_enabled": False}
```

Comment:
we started a new paradigm that I think makes tests more efficient, if we only want to test just the null_count info, we can pretty much disable everything else:
```
{"row_statistics.null_count.is_enabled": False}
```
becomes,
```
{
    "*.is_enabled": False,
    "row_statistics.*.is_enabled": True,
    "row_statistics.null_count.is_enabled": False,  # or True for the other case
}
```

Reply:
I believe in this case we won't need all the mocks above either. If you run this test specifically, i'd expect times in the `miliseconds`

Reply:
+1

Reply:
yep told @drahc1R to test on the mocks if we need those... didn't think we would need them